src/test/java/com/xceptance/neodymium/tests/NeodymiumTest.java

```diff
@@ -120,4 +204,4 @@ public static void writeMapToPropertiesFile(Map<String, String> map, File file)
             throw new RuntimeException(e);
         }
```

Comment:
Why not using what you have at hand: FileUtils from Apache commons-io library has all you need.
```java
  import org.apache.commons.io.FileUtils;
  import java.nio.charset.StandardCharsets;
  FileUtils.writeStringToFile(propertiesString, file, StandardCharsets.UTF-8);
```

Reply:
already done in development, but was not pushed :-(
I improved this on #131 since the change originally was done there.